examples/src/withConfig.js

```diff
@@ -10,12 +11,13 @@ function withConfig(WrappedComponent) {
         render() {
             return (
                 <WrappedComponent
-                    {...this.props}
+                    features={features}
                     fileId={FILE_ID}
                     language="en-US"
                     messages={messages}
                     rootFolderId={FOLDER_ID}
                     token={TOKEN}
+                    {...this.props}
```

Comment:
Do all of the examples still work after this change?

Reply:
yes, none of them actually pass props